.ado/jobs/universal.yml

```diff
@@ -108,7 +113,12 @@
                       Mso.UnitTests\**
 
             - job: UniversalTest${{ matrix.Name }}
-              variables: [template: ../variables/windows.yml]
+              variables:
+                - template: ../variables/windows.yml
+                # Some tasks run on a different user (VssAdministrator) instead of the default user (AzDevOps).
+                # Keep NuGet cache independent from the user directory.
+                - name: NUGET_PACKAGES
+                  value: D:\NugetPackages
```

Comment:
Can you clarify what "other machines"?

I kept it because got an inconsistent restore path between two types of hosts we were using (I don't know if the agents are all homogeneous now). It's still a safe practice, in case we have mixed machines for different jobs/phases.

I'll update the value to use the Agent's TEMP as suggested.